Supports the use of multiple template rendering engines in one application. (#8673)
abpframework/abp: docs/en/Text-Templating.md

* **[Razor](Text-Templating-Razor.md)**
* **[Scriban](Text-Templating-Scriban.md)**

> You can use different template engines at the same time, or even create a new template engine.

Reviewer: ```suggestion
You can use different template engines in the same application, or even create a new custom template engine.
```
